dialogs/measurement/depth_profile_numeric_limits.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+"""
+Created on 21.3.2013
+Updated on 30.5.2018
+
+Potku is a graphical user interface for analyzation and
+visualization of measurement data collected from a ToF-ERD
+telescope. For physics calculations Potku uses external
+analyzation components.
+Copyright (C) 2013-2018 Jarkko Aalto, Severi Jääskeläinen, Samuel Kaiponen,
+Timo Konu, Samuli Kärkkäinen, Samuli Rahkonen, Miika Raunio, Heta Rekilä and
+Sinikka Siironen, 2020 Juhani Sundell, Tuomas Pitkänen
```

Comment:
If we hold on to previous copyright ownership practices, this should be changed to `Copyright (C) 2021 n.n.`